shell/ui/styles.css

```
@@ -1238,66 +1247,73 @@
-  justify-content: center;
+[data-iii-ui="shell"] .shui-review-toolbar .active {
+  color: var(--color-ink);
+  background: var(--color-surface-hover);
+}
+[data-iii-ui="shell"] .shui-diff-style-icon {
+  width: 16px;
+  height: 16px;
+}
+[data-iii-ui="shell"] .shui-diff-style-icon .frame {
+  fill: none;
+  stroke: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)